script/main.js

```diff
@@ -44,18 +44,14 @@ async function showNotebook(){
     const notebook = await fetchNotebooks();
 
     for(let i = 0; i < notebook.length; i++){
```

Comment:
voor data die van database komen zou ik liever foreach gebruiken,

dan hoef je ook geen ```notebook[i].id``` te gebruiken maar kan je gwn ```item.id``` gebruiken in de plaats